main.py

```diff
@@ -0,0 +1,35 @@
+import argparse
+from sourcecode.driver import Driver
```

Comment:
Imports should be categorized into three secitons
```
built in imports

third party imports

local imports
```

Refactor imports accordingly.